src/promdb2pcp/promdb2pcp.py

```
@@ -0,0 +1,601 @@
+    log = pmi.pmiLogImport(archive_name)
+
+    meta_file = os.path.join(data_dir, 'metadata.json')
+    if os.path.exists(meta_file):
+        with open(meta_file) as f:
+            meta = json.load(f)
+        host = meta.get('node', hostname)
+        log.pmiSetHostname(host)
+        tz_str = meta.get('timezone', 'UTC')
+        try:
+            log.pmiSetTimezone(tz_str)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)